### ISSUE TITLE
Create Man Page

### ISSUE THREAD

**Issue by TekWizely** [TekWizely/run#23]
If Run had a proper man page, it would feel more like an official *nix tool.

**Comment by TekWizely** [comment]
@geoelectric Thinking this might a task you could help with?

**Comment by geoelectric** [comment]
Yeah, quite possibly! Let me do a little research this weekend and see what's involved. I'm kind of hoping I don't have to learn groff, but I imagine there's boilerplate out there.

If it's easy enough, I'll at least do a PR with a placeholder that you can update with real text.
